README.adoc

```diff
@@ -58,13 +59,17 @@ to create, run, test, and deploy an application built from a Kabanero Collection
 
 == Discovering Appsody repositories and stacks
 
-Appsody is an intergal component of Kabanero, its designed to enable applications to be built and tested inside a docker container, which then can be deployed by Kabanero into a target host environment.  
+Appsody is an integral component of Kabanero, designed to enable applications to be built and tested inside a docker container, which can then
```

Comment:
I personally don't like the phrase "Appsody is an integral component of Kabanero" as you could read this as us defining what Appsody is and yet Appsody has no dependency on Kabanero.

docker container -> Docker container

Reply:
How about:

"Kabanero contains Appsody, which is designed to enable applications to be built and tested inside a Docker container. When built, the Docker container can be deployed by Kabanero into a target host environment"

@ralanlittle - modification to your input, so please comment.